lib/faraday/encoders/flat_params_encoder.rb

```diff
@@ -47,7 +47,7 @@ def self.encode(params)
 
     def self.decode(query)
       empty_accumulator = {}
-      return nil if query == nil
+      return nil if query.nil?
```

Comment:
Could this be the first line of this method?

Reply:
yes, agree